src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/Et1VettingService.java

```diff
@@ -110,6 +111,114 @@ public void populateHearingVenue(CaseData caseData) {
+        caseData.setVettingJurisdictionCodeCollection(null);
+
+        caseData.setIsTrackAllocationCorrect(null);
+        caseData.setSuggestAnotherTrack(null);
+        caseData.setWhyChangeTrackAllocation(null);
+        caseData.setTrackAllocationGeneralNotes(null);
+        caseData.setIsLocationCorrect(null);
+        caseData.setWhyChangeOffice(null);
+        caseData.setEt1LocationGeneralNotes(null);
+        caseData.setTrackAllocation(null);
+        caseData.setTribunalAndOfficeLocation(null);
+        caseData.setRegionalOffice(null);
+        caseData.setRegionalOfficeList(null);
+        caseData.setEt1AddressDetails(null);
+        caseData.setEt1TribunalRegion(null);
+        caseData.setEt1HearingVenues(null);
+        caseData.setEt1SuggestHearingVenue(null);
+        caseData.setEt1HearingVenueGeneralNotes(null);
+        caseData.setEt1GovOrMajorQuestion(null);
+
+        caseData.setEt1ReasonableAdjustmentsQuestion(null);
+        caseData.setEt1ReasonableAdjustmentsTextArea(null);
+        caseData.setEt1VideoHearingQuestion(null);
+        caseData.setEt1VideoHearingTextArea(null);
+        caseData.setEt1FurtherQuestionsGeneralNotes(null);
+
+        caseData.setReferralToJudgeOrLOList(null);
+        caseData.setAclaimOfInterimReliefTextArea(null);
+        caseData.setAstatutoryAppealTextArea(null);
+        caseData.setAnAllegationOfCommissionOfSexualOffenceTextArea(null);
+        caseData.setInsolvencyTextArea(null);
+        caseData.setJurisdictionsUnclearTextArea(null);
+        caseData.setLengthOfServiceTextArea(null);
+        caseData.setPotentiallyLinkedCasesInTheEcmTextArea(null);
+        caseData.setRule50IssuesTextArea(null);
+        caseData.setAnotherReasonForJudicialReferralTextArea(null);
+        caseData.setEt1JudgeReferralGeneralNotes(null);
+        caseData.setReferralToREJOrVPList(null);
+        caseData.setVexatiousLitigantOrderTextArea(null);
+        caseData.setAnationalSecurityIssueTextArea(null);
+        caseData.setNationalMultipleOrPresidentialOrderTextArea(null);
+        caseData.setTransferToOtherRegionTextArea(null);
+        caseData.setServiceAbroadTextArea(null);
+        caseData.setAsensitiveIssueTextArea(null);
+        caseData.setAnyPotentialConflictTextArea(null);
+        caseData.setAnotherReasonREJOrVPTextArea(null);
+        caseData.setEt1REJOrVPReferralGeneralNotes(null);
+        caseData.setOtherReferralList(null);
+
+        caseData.setClaimOutOfTimeTextArea(null);
+        caseData.setMultipleClaimTextArea(null);
+        caseData.setEmploymentStatusIssuesTextArea(null);
+        caseData.setPidJurisdictionRegulatorTextArea(null);
+        caseData.setVideoHearingPreferenceTextArea(null);
+        caseData.setRule50IssuesForOtherReferralTextArea(null);
+        caseData.setAnotherReasonForOtherReferralTextArea(null);
+        caseData.setEt1OtherReferralGeneralNotes(null);
+        caseData.setEt1VettingAdditionalInformationTextArea(null);
```

Comment:
I had referrals in my mind where we assign values to referralCollection and then clear caseData fields. I have now just markup fields cleared, just wonder though, if someone goes back to Et1Vetting tab, won't the markups be missing?

Reply:
We don't display the markup in the tabs. Just the information that's entered. I've seen the refactored changes but there are more fields that would potentially need setting to null. The screenshot is all the potential markdown we have on the event so it might be worth cross referencing this to the event to see if it is markdown or not
![Screenshot 2023-09-20 at 09 21 09](https://github.com/hmcts/et-ccd-callbacks/assets/78368309/6dfa2b87-5b72-453b-92da-e4d5cc63430b)